cmd/e3db/main.go

```diff
@@ -354,7 +354,7 @@ func cmdDelete(cmd *cli.Cmd) {
 		client := options.getClient()
 
 		for _, recordID := range *recordIDs {
-			err := client.Delete(context.Background(), recordID)
+			err := client.Delete(context.Background(), recordID, "")
```

Comment:
Should actually use the safe delete, no?

Reply:
In the CLI, `delete` just takes a record ID ... unless we extend it to also require a version ID (which you'd have to somehow get through another method), then safe delete wouldn't work. For a simple CLI, skipping the safe delete feels OK, but I'm willing to discuss.